src/main/java/com/wcc/platform/configuration/DatabaseConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.wcc.platform.configuration;
+
+import com.surrealdb.connection.SurrealWebSocketConnection;
+import com.surrealdb.driver.SyncSurrealDriver;
+import com.wcc.platform.repository.surrealdb.SurrealDbConfig;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/** Database configuration to initialize Driver and connection. */
+@Configuration
+public class DatabaseConfig {
+
+  /** Create Sync driver connection for SurrealDB. */
+  @Bean
+  public SyncSurrealDriver getDriver(final SurrealDbConfig config) {
+    final var conn =
+        new SurrealWebSocketConnection(config.getHost(), config.getPort(), config.isTls());
+    conn.connect(config.getConnections());
+    final var driver = new SyncSurrealDriver(conn);
+    driver.use(config.getNamespace(), config.getDatabase());
+
+    return driver;
```

Comment:
TODO: Check about this -> Closing the Connection: It’s important to close the driver connection once you're done to avoid resource leaks.
driver.close() or conn.disconnect()

Reply:
You are complete right, I should not been creating the connection as a Bean because we are going to keep the connection open from start to the end. I will keep this version like this and I will take care of this change in another PR